packages/grafana-ui/src/components/TimePicker/time.ts

```diff
@@ -1,43 +1,41 @@
-import { TimeOption, TimeRange, TIME_FORMAT } from '../../types/time';
+import { TimeOption, TimeRange, TIME_FORMAT, RawTimeRange, TimeZone } from '../../types/time';
 import { describeTimeRange } from '../../utils/rangeutil';
 import * as dateMath from '../../utils/datemath';
 import { dateTime, DateTime, toUtc } from '../../utils/moment_wrapper';
 
-export const mapTimeOptionToTimeRange = (
-  timeOption: TimeOption,
-  isTimezoneUtc: boolean,
-  timezone?: dateMath.Timezone
-): TimeRange => {
-  const fromMoment = stringToDateTimeType(timeOption.from, isTimezoneUtc, false, timezone);
-  const toMoment = stringToDateTimeType(timeOption.to, isTimezoneUtc, true, timezone);
-
-  return { from: fromMoment, to: toMoment, raw: { from: timeOption.from, to: timeOption.to } };
+export const mapTimeOptionToTimeRange = (timeOption: TimeOption, timeZone?: TimeZone): TimeRange => {
+  const fromMoment = stringToDateTimeType(timeOption.from, false, timeZone);
+  const toMoment = stringToDateTimeType(timeOption.to, true, timeZone);
```

Comment:
Nitpick, but would be nice to remove any moment references

s/fromMoment/fromDateTime
s/toMoment/toDateTime